g2p/mappings/langs/generated/str-ipa_to_eng-ipa.json

```diff
@@ -8,11 +8,11 @@
         "out": "dz"
     },
     {
-        "in": "e",
-        "out": "e\u02d0"
+        "in": "e\u02d0",
```

Comment:
What is the purpose of transducing `e\u02d0 -> e` only to transduce it back from `e -> e\u02d0` in the next rule? It looks like you're not preventing feeding in the config and the rules are applied as-is, so are these two rules needed at all?

Reply:
Oh, this is for input of `e\u02d0` from the previous mapping - I see. You're trying to prevent `eːː`?

Reply:
yes exactly! It's a bit of a hack-y fix I know, but when we went over it with Eric, he said the issue was in the generate-mapping function and that this was a good enough fix to just get str working until the bug gets resolved in the actual function